pages/docs/migration/migrating-to-v3.md

```diff
@@ -0,0 +1,393 @@
+
+```yml
+asyncapi: 2.6.0
+...
+channels:
+  user/signedup:
+    message: 
+      oneOf:
+        - ...
+        - ...
+
+asyncapi: 2.6.0
+...
+channels:
+  user/signedup:
+    message: 
+      ...
+```
+
+```yml
+asyncapi: 3.0.0
+...
+channels:
+  UserSignup:
+    address: user/signedup
+    messages: 
+      UserMessage: 
+        ...
+      UserMessage2:
+        ...
+
+asyncapi: 3.0.0
+...
+channels:
+  UserSignup:
+    address: user/signedup
+    messages: 
+      UserMessage: 
+        ...
+```
+
+### Unifying explicit and implicit references
+
+In v2, it was possible to do implicit references in some places, for example for server security configuration, it was by name which referred to a [Security Schema Object](https://www.asyncapi.com/docs/reference/specification/v2.6.0#securitySchemeObject) in components - And for a channel to reference global servers by name.
+
+In v3, this information MUST be explicit references. This did mean that we had to slightly change the [Server Object](https://www.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#serverObject) `security` property, which is now an array instead of an object. We then moved the information about needed scopes for OAuth and OpenID Connect to the [Security Scheme Object](https://www.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#securitySchemeObject).
+
+```yml
+asyncapi: 2.6.0
+servers:
+  production:
+    ...
+    security:
+      oauth_test: ["write:pets"]
+...
+channels: 
+  test/path:
+    severs:
+      - production
+components:
+  securitySchemes:
+    oauth_test: 
+      type: oauth2
+      flows:
+        implicit:
+          authorizationUrl: https://example.com/api/oauth/dialog
+          availableScopes:
+            write:pets: modify pets in your account
+            read:pets: read your pets
+      scopes:
+        - 'write:pets'
+```
+
+```yml
+asyncapi: 3.0.0
+servers:
+  production:
+    ...
+    security:
+      - $ref: "#/components/securitySchemes/oauth_test"
+...
+channels: 
+  test/path:
+    severs:
+      - $ref: "#/servers/production"
+components:
+  securitySchemes:
+    oauth_test:
+      type: oauth2
+      flows:
+        implicit:
+          authorizationUrl: https://example.com/api/oauth/dialog
+          availableScopes:
+            write:pets: modify pets in your account
+            read:pets: read your pets
+      scopes:
+        - "write:pets"
+```
+
+### New trait behavior
+Traits in v2 always replaced any duplicate properties that were defined both in traits and the associated object. This meant for example if the message traits defined headers and the message object did as well, only the message trait headers would be applied because it overwrote anything you wrote in the Message Object.
```

Comment:
```suggestion
Traits in v2 always replaced any duplicate properties that were defined both in traits and the associated object. This meant for example, if the message traits defined headers and the message object did as well, only the message trait headers would be applied because it overwrote anything you wrote in the Message Object.
```